libgeopmd/include/geopm_public.h

```diff
@@ -14,3 +14,17 @@
  */
 #define GEOPM_PRIVATE __attribute__((visibility("hidden")))
 #endif
+
+/*
+ * String length allocated for GEOPM names (like signal and control
+ * names). Has same value as NAME_MAX from limits.h for historical
+ * reasons.
+ */
+#define GEOPM_NAME_MAX 255
```

Comment:
If it's mainly for signal/control names, should this go in geopm_pio.h (or maybe a separate header used by all signal/control interfaces (I guess that's just pio and stats_collector right now)?